cs/src/core/expressions/ObjectParser.cs

```diff
@@ -134,6 +134,14 @@ Expression Field(ITransform transform, Expression structVar, UInt16 id, ISchemaF
             if (fieldSchemaType.IsBondStruct() || fieldSchemaType.IsBonded() || schemaField.GetModifier() != Modifier.Optional)
```

Comment:
This bug also affects bonded and the other modifiers that are not optional.

Reply:
struct fields cannot be aliased, nor can bonded fields, so really is just affects blob fields that aren't optional.